### ISSUE TITLE
Defunct?

### ISSUE THREAD

**Issue by konklone** [unitedstates/data-issues#7]
Should we change the repo description to mark this as unmaintained?

**Comment by JoshData** [comment]
Maybe we can export the issues data and store it statically somewhere else, then delete this repo?

**Comment by konklone** [comment]
Sounds like the right call in theory, just not sure of an easy way to do that...

**Comment by JoshData** [comment]
Fair enough. (There are only 6 issues so copy-paste would work but I'm not sure where we'd put the file.)

In the meanwhile, :+1: to renaming or changing the description.
